modin/tests/pandas/dataframe/test_join_sort.py

```diff
@@ -230,20 +230,20 @@ def test_join_6602():
     "test_data, test_data2",
     [
         (
-            np.random.uniform(0, 100, size=(2**6, 2**6)),
-            np.random.uniform(0, 100, size=(2**7, 2**6)),
```

Comment:
The problem is that the left merge can change the shape of the result if the merge occurs using values that are in both the left and right operands. However, in our testing we used float numbers, which were not considered identical when compared and the problem was not visible.